deploy.go

```diff
@@ -106,18 +106,19 @@ func (p *deploycmd) scan(c *cli.Context) error {
 			return nil
 		}
 
-		e := p.deploy(c, path)
+		err = p.deploy(c, path)
 		if err != nil {
-			fmt.Println(path, e)
+			// fmt.Println(path, e)
+			return fmt.Errorf("deploy error on %s: %v", path, err)
 		}
 
 		now := time.Now()
 		os.Chtimes(path, now, now)
 		walked = true
-		return e
+		return err
```

Comment:
if deploy returns a nil error (which we want) then this will walk the rest of the contents under `wd` (i assume this means working directory) even though we have already called `deploy`.

it would make sense to just use the `walked` value and use a sentinel error to return from `filepath.Walk` to indicate that we found the path and to not walk the rest of the directory's contents. again, i could imagine this taking a long time if a user does this in a directory like `$HOME` or in a really large repo.

Reply:
actually, due to this i wonder if it's possible that `deploy` gets called twice, on two different func files? eesh, need to stop that if that's true.

Reply:
Yes, this is remnants of old crap, it looks for all funcfiles recursively and deploys them all. Thought of removing it since we only do singles for now, but I know I'll want it again when we have full app deploys.

Reply:
ahhh, makes sense.